app/renderer/components/common/dialog.js

```diff
@@ -42,8 +42,7 @@ class Dialog extends ImmutableComponent {
 
   render () {
     return <div className={cx({
-      [css(styles.dialog)]: true,
-      [css(styles.dialog_isNotClickDismiss)]: !this.props.isClickDismiss,
+      [css(styles.dialog, !this.props.isClickDismiss && styles.dialog_isNotClickDismiss)]: true,
```

Comment:
this avoids parsing styles twice which is ok but still a nit win as it parses again every time a `className` with `css()` method is called.

What I'm not confident is that it makes the code less readable and error prone IMHO. `cx()` will eventually be replaced so I don't think it offers much value. This is just my take so please lmk if you disagree.